sdk/core/System.ClientModel/src/ModelReaderWriter/ModelWriterOfT.SequenceBuilder.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Buffers;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Primitives
+{
+    internal partial class ModelWriter<T> : IDisposable
```

Comment:
Should it go in the `System.ClientModel.Internal` namespace following the convention in the feature branch?